test/csv/test_csv_filter.rb

```diff
@@ -137,4 +137,19 @@ def test_option_col_sep
     assert_equal(["aaa.bbb.ccc\nddd.eee.fff\n", ""],
                  run_csv_filter(csv, "--col-sep=:", "--output-col-sep=."))
   end
+
+  def test_option_quote_char
+    quote_char = "'"
+    csv = CSV.generate(quote_char: quote_char) do |csv|
+      csv << ['foo', 0]
+      csv << ["'bar'", 1]
+      csv << ['"baz"', 2]
+    end
+    assert_equal(["foo,0\n'''bar''',1\n\"baz\",2\n", ""],
+                 run_csv_filter(csv, "--quote-char=:"))
+    assert_equal(["foo,0\n'''bar''',1\n\"\"\"baz\"\"\",2\n", ""],
+                 run_csv_filter(csv, "--input-quote-char=:"))
```

Comment:
Should we specify `--quote-char=...` too?

Reply:
I don't understand.  Doesn't the default value serve us well enough?

Reply:
`--row-sep` tests use `--row-sep` and `--{input,output}-row-sep`.
I think that we want to test that `--quote-char` configuration is overridden by `--{input,output}-quote-char` here. Is it incorrect?

If we want to test only `--{input,output}-quote-char`, we should do it in the test for `--{input,output}-quote-char` not the test for `--quote-char`.